src/main/java/net/daporkchop/fp2/asm/core/server/IMixinWorldBorder.java

```diff
@@ -0,0 +1,14 @@
+package net.daporkchop.fp2.asm.core.server;
+
+import net.minecraft.world.border.WorldBorder;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.gen.Accessor;
+
+@Mixin(WorldBorder.class)
```

Comment:
the current `master` branch is still using Forge access transformers instead of mixin accessors, please use that instead. (`META-INF/fp2_at.cfg` in resources)